doc/napi_ids.txt

```diff
@@ -0,0 +1,37 @@
+NAPI ID based worker thread selection
+  -N <num_napi_ids>  | --napi_ids=<num_napi_ids>
+
+By default memcached assigns connections to worker threads in a round-robin
+manner. NAPI ID based worker thread selection enables worker threads to be
+selected based on the NIC HW RX queue on which the incoming request is
+received.
+
+This is enabled using SO_INCOMING_NAPI_ID socket option that is supported
```

Comment:
This socket option doesn't seem to be part of the socket man page. Any plan on filling in the gap there?

Reply:
Yes. looks like we missed submitting this update to the socket man page Will do soon.

Reply:
Done. Submitted the update and you can see the commit here.
  socket.7: Document SO_INCOMING_NAPI_ID
  https://github.com/mkerrisk/man-pages/commit/e8500ecc786f5ee5854d4aa810dcf5124e8ed23f